cmd/root.go

```diff
@@ -12,16 +12,19 @@ import (
 	"github.com/numaproj-labs/numaflow-perfman/util"
 )
 
-var config *rest.Config
-var kubeClient *kubernetes.Clientset
-var dynamicClient *dynamic.DynamicClient
-var log *zap.Logger
+var (
+	config        *rest.Config
+	kubeClient    *kubernetes.Clientset
+	dynamicClient *dynamic.DynamicClient
+	log           *zap.Logger
+)
 
 // rootCmd represents the base command when called without any subcommands
 var rootCmd = &cobra.Command{
-	Use:   "perfman",
-	Short: "Numaflow performance testing framework",
-	Long:  "Perfman is a command line utility for performance testing changes to the numaflow platform",
+	Use:          "perfman",
+	Short:        "Numaflow performance testing framework",
+	Long:         "Perfman is a command line utility for performance testing changes to the numaflow platform",
+	SilenceUsage: true,
```

Comment:
just curious, what does SilenceUsage mean?

Reply:
I added this flag to disable the behaviour of showing the help menu whenever there is an error., since it takes away from the error message and the error didn't occur because user doesn't know how to use perfman but because of something else